src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -110,6 +111,19 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
             // If "ptrace_scope" is a value other than 0, only root can attach to arbitrary processes
             bool requiresRootAttach = this.GetPtraceScope() != 0;
 
+            // Check and see if gnome-terminal exists. If not, fall back to xterm
+            string terminalCmd, bashCommandPrefix;
+            if (File.Exists(GnomeTerminalPath))
+            {
+                terminalCmd = GnomeTerminalPath;
+                bashCommandPrefix = "--title DebuggerTerminal -x";
+            }
+            else
```

Comment:
Probably, but right now it just says that gnome-terminal (or xterm) isn't available